File: exercicios/1019/ct.pas
program ct (input, output);
Uses sysutils;

function timming (n: LongInt): LongInt;
begin
    if(n > 60) then  
        timming := timming (n mod 60)
    else
        timming := (n mod 60)  
end;
var
    n, s, m, h: LongInt;

begin
    readLn(n);
    s := timming(n);
    m := timming (n div 60);
    h := timming (n div 3600) ; 
    writeln (h,':',m,':',s );   
end.

File: exercicios/1019/ct.ml
let rec timming n =
  if n > 60 then timming (n mod 60)
  else n mod 60;;

let n = read_int();;
let s = timming n;;
let m = timming (n / 60);;
let h = timming (n / 3600);;
Printf.printf "%d:%d:%d\n" h m s;;